### PR TITLE
Add XPU support for SpeedySpeech

### PR DESCRIPTION
**Pull request by USTCKAY**
### PR types

New features
### PR changes

Others
### Describe

add xpu support for SpeedySpeech

### CODE REVIEW

File: paddlespeech/t2s/exps/synthesize_e2e.py [hunk 1]
```diff
@@ -329,12 +334,14 @@ def parse_args():
 def main():
     args = parse_args()
 
-    if args.ngpu == 0:
+    if args.ngpu > 0:
+        paddle.set_device("npu")
+    elif args.xgpu > 0:
+        paddle.set_device("xpu")
+    elif args.ngpu == 0 and args.nxpu == 0:
         paddle.set_device("cpu")
```
Comment (zxcd) [inline paddlespeech/t2s/exps/synthesize_e2e.py:343]: gpu?

File: paddlespeech/t2s/exps/synthesize.py [hunk 2]
```diff
@@ -219,7 +219,12 @@ def parse_args():
     )
     # other
     parser.add_argument(
-        "--ngpu", type=int, default=1, help="if ngpu == 0, use cpu.")
+        "--ngpu", type=int, default=1, help="if ngpu == 0, use cpu or xpu.")
+    parser.add_argument(
+        "--nxpu",
```
Comment (zxcd) [inline paddlespeech/t2s/exps/synthesize.py:224]: xpu的使用逻辑最好写的清晰一些。

Comment (USTCKAY) [reply paddlespeech/t2s/exps/synthesize.py:224]: done

File: paddlespeech/t2s/exps/synthesize_e2e.py (hunk 1 above)
Comment (USTCKAY) [reply paddlespeech/t2s/exps/synthesize_e2e.py:343]: 已修改拼写错误

**Review by zxcd — APPROVED** [review]
LGTM

**Pull request merged**